templates/footer.html

```diff
@@ -0,0 +1,5 @@
+				</div>
```

Comment:
Better, but I still think `footer.html` is not needed. Although now I must agree that it *is* reusable.. :P 

I know what you're thinking: include `.js` files in the end of HTML in an easy way. 

The way I personally think it would be best is:

`header.html`: All the code inside `<head></head>` but *NOT* the actual `<head>` (so a single page can have an additional CSS). 
`navbar.html`: All the code needed to create the Navigation Bar, without of course the Stylesheets.
`footer.html`: An empty (currently) file that's included in the bottom of every page (that needs one). 
`index.html` / `list.html` / ... : Everything else to create the same page.